src/datasets/load.py

```diff
@@ -1112,14 +1110,11 @@ def dataset_module_factory(
             _raise_if_offline_mode_is_enabled()
             hf_api = HfApi(config.HF_ENDPOINT)
```

Comment:
I think you can remove this line because `hf_api` is no longer used in this method?
```python
hf_api = HfApi(config.HF_ENDPOINT)
```